README.md

```diff
@@ -46,6 +46,44 @@ sure to install the new TALs using
 routinator init --force
 ```
 
+## Quick Start with Pre-built Packages
```

Comment:
Perhaps instead of ‘Pre-built Packages’ specifically saying ’Debian and Ubuntu’ would be better?